### DISCUSSION TITLE
pre-install questions?

### DISCUSSION THREAD

**Discussion by Remarkable-Guille** [postalserver/postal#2734] [Help with using Postal]
Hi, 

A couple of questions about Postal:

1. Can it be integrated with third-party SMTP services like Mailchannels?
2. Is multi-tenant? I read in the docs that it can host multiple organizations. Can each organization/user have access to their panel and stats?

Thank You

**Comment by Pacerino** [comment]
1. You can forward incoming messages to another smtp server or an e-mail address. However, I think that Postal cannot send via mail channels.
2. You can create organisations and assign users to the respective organisation.
![image](https://github.com/postalserver/postal/assets/22194063/83a0f97c-320b-4292-bf6f-dae044f4462a)

**Reply by Remarkable-Guille** [reply]
Thanks for your reply!

If forwarding to another SMTP is possible, It can be used as a delivery gateway to MailChannels.
Postal > Posfix > MailChannels

Time to install Postal and see how it goes

**Reply by Pacerino** [reply]
But only for incoming messages! If your application is sending a mail to Postal, it cant be forwarded to another server.
Outgoing: Application -> Postal -> World
Incoming: World -> Postal -> Postfix -> MailChannels

Cheers!
